legend-engine-pure-code-compiled-core/src/main/resources/core/pure/executionPlan/javaPlatform/executionNode/graphFetch/inMemory/graphFetchInMemory.pure

```diff
@@ -135,12 +135,15 @@ function <<access.private>> meta::pure::executionPlan::engine::java::graphFetch:
             let graphImpl      = $node->graphImplClass($path, $context);
             let currentSet     = $graphFetchTree.sets->toOne()->cast(@InstanceSetImplementation);
             let input          = j_parameter(javaObject(), 'input');
+            let globalObject   = javaField(['public'], javaHashMap(javaObject(),javaObject()), 'sharedObject',javaHashMap()->j_new([]));
```

Comment:
Lets add a comment that this field is deprecated and should not be used

Reply:
thank you, sure